auth.go

```diff
@@ -68,7 +79,7 @@ func (b *mesosBackend) pathLogin(ctx context.Context, req *logical.Request, d *f
 	return &logical.Response{
 		Auth: &logical.Auth{
 			Policies: policies,
-			Period:   10 * time.Minute,
+			Period:   cfg.TTL,
```

Comment:
Can this be `cfg.Period`? I know the AWS auth backend uses a field called `period` (in addition to `ttl` and `max_ttl`).

Reply:
It most certainly can. :-)